pr2eus_tutorials/launch/pr2_tabletop.launch

```diff
@@ -5,11 +5,25 @@
   <arg name="manager" default="pr2_tabletop_object_detector_nodelet_manager"/>
   <arg name="machine" default="c2" />
   <arg name="load_machine" default="false" />
+  <arg name="remote" default="false" />
+
+  <arg name="RGB_CAMERA_INFO" value="/kinect_head/rgb/camera_info" />
+  <arg name="RGB_IMAGE" value="/kinect_head/rgb/image_rect_color" />
+  <arg name="RGB_IMAGE_REMOTE" value="/kinect_head_remote/rgb/image_rect_color" />
+  <arg name="DEPTH_CAMERA_INFO" value="/kinect_head/depth_registered/camera_info" />
+  <arg name="DEPTH_IMAGE" value="/kinect_head/depth_registered/image_rect" />
+  <arg name="DEPTH_IMAGE_REMOTE" value="/kinect_head_remote/depth_registered/image_rect" />
+  <arg name="PROSILICA_CAMERA_INFO" value="/prosilica/camera_info" />
+  <arg name="PROSILICA_IMAGE" value="/prosilica/image_raw" /> 
+  <arg name="PROSILICA_IMAGE_REMOTE" value="/prosilica_remote/image_raw" /> 
+  <arg name="POINT_CLOUD" value="/kinect_head/depth_registered/throttled/points" />
+  <arg name="POINT_CLOUD_REMOTE" value="/kinect_head_remote/depth_registered/points" />
 
   <include file="$(find pr2_machine)/$(env ROBOT).machine" unless="$(arg load_machine)" />
```

Comment:
We do not need this line if we use remote?

Reply:
Thank you, you mean summing up remote topics with <group> tag is more readable ?

Reply:
Or you mention L22?

Reply:
> Or you mention <include file="$(find pr2_machine)/$(env ROBOT).machine" unless="$(arg load_machine)" /> ?

Yes, I mention this line.

Reply:
OK, fix it by [791eb8682f830106bf868e7405cd555da150ab2a](https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/791eb8682f830106bf868e7405cd555da150ab2a)

Reply:
Ok, fix by https://github.com/jsk-ros-pkg/jsk_pr2eus/pull/459/commits/791eb8682f830106bf868e7405cd555da150ab2a